src/main/webapp/inc.modul.jsp

```diff
@@ -5,1768 +5,1932 @@
+                            + "</td>");
+
+                    String vonTag = rs.getString("h.VonTag");
+                    String vonMonat = rs.getString("h.VonMonat");
+                    String vonJahr = rs.getString("h.VonJahr");
+                    String vonJhdt = rs.getString("h.VonJahrhundert");
+                    String bisTag = rs.getString("h.BisTag");
+                    String bisMonat = rs.getString("h.BisMonat");
+                    String bisJahr = rs.getString("h.BisJahr");
+                    String bisJhdt = rs.getString("h.BisJahrhundert");
+
+                    String von = "";
+
+                    if (vonTag != null && !vonTag.equals("")
+                            && !vonTag.equals("0")) {
+                        von = vonTag + ".";
+                    }
+                    if (vonMonat != null && !vonMonat.equals("")
+                            && !vonMonat.equals("0")) {
+                        von = von + vonMonat + ".";
+                    }
+                    if (vonJahr != null && !vonJahr.equals("")
+                            && !vonJahr.equals("0")) {
+                        von = von + vonJahr;
+                    }
+                    if (von.equals("") && vonJhdt != null) {
+                        von = vonJhdt;
+                    }
+
+                    String bis = "";
+
+                    if (bisTag != null && !bisTag.equals("")
+                            && !bisTag.equals("0")) {
+                        bis = bisTag + ".";
+                    }
+                    if (bisMonat != null && !bisMonat.equals("")
+                            && !bisMonat.equals("0")) {
+                        bis = bis + bisMonat + ".";
+                    }
+                    if (bisJahr != null && !bisJahr.equals("")
+                            && !bisJahr.equals("0")) {
+                        bis = bis + bisJahr;
+                    }
+                    if (bis.equals("") && bisJhdt != null) {
+                        bis = bisJhdt;
+                    }
+
+                    if (!bis.equals(von) && !bis.equals("")) {
+                        out.println("<td class=\"ut-table__item ut-table__body__item\">" + von + " - " + bis
+                                + "</td>");
+                    } else {
+                        out.println("<td class=\"ut-table__item ut-table__body__item\">" + von + "</td>");
+                    }
+
+                    String bem = rs.getString("et.Bemerkung");
+
+                    if (bem == null) {
+                        bem = "";
+                    }
+
+                    out.println("<td class=\"ut-table__item ut-table__body__item\">" + bem + "</td>");
+                    out.println("</tr>");
+                }
+            } catch (Exception e) {
+                out.println(e);
+            } finally {
+                try {
+                    if (null != rs) {
```

Comment:
Unfortunately inc.modul.jsp is still managing its own SQL connections, but this might be out of scope for the current PR (we should not forget about this anyway).

Reply:
will be done in another pr, i wrote it to my to-do list